Source/Data/EventAnalysis/KTSparseWaterfallCandidateData.hh

```diff
@@ -25,7 +25,10 @@ namespace Katydid
                 double fFrequency;
                 double fAmplitude;
                 double fTimeInAcq;
-                Point(double tirc, double freq, double amp, double tiacq) : fTimeInRunC(tirc), fFrequency(freq), fAmplitude(amp), fTimeInAcq(tiacq) {}
+                double fMean;
+                double fVariance;
+                double fNeighborhoodAmplitude;
+                Point(double tirc, double freq, double amp, double tiacq, double mean, double variance, double neighborhoodAmplitude) : fTimeInRunC(tirc), fFrequency(freq), fAmplitude(amp), fTimeInAcq(tiacq), fMean(mean), fVariance(variance), fNeighborhoodAmplitude(neighborhoodAmplitude) {}
             };
 
```

Comment:
The SparseWaterfallCandidateData will have more variables (for example MinimumTimeInRunC aso) in the future. They don't exist in this branch yet but once they get merged, the DBScanTrackClustering will have to set those too. I don't think that that needs to hold up this pull request. This project is very interleaved and currently I have no suggestion how to disentangle the changes more than we are doing now.